bcda/models/models.go

```diff
@@ -84,14 +84,26 @@ func (job *Job) CheckCompletedAndCleanup() (bool, error) {
 	if int(completedJobs) >= job.JobCount {
 
 		staging := fmt.Sprintf("%s/%d", os.Getenv("FHIR_STAGING_DIR"), job.ID)
-		err := os.Remove(staging)
+		payload := fmt.Sprintf("%s/%d", os.Getenv("FHIR_PAYLOAD_DIR"), job.ID)
```

Comment:
Are these changes from a different PR?  I don't see how encryption removal would affect this stuff.  Maybe you need to sync with `master`?

Reply:
yep this change is being pulled from the BCDA-1591 pr, see the Change Details for more details. Once the other pr is merged, this one will resolve those differences with a pull from master. But  should be done after the other one is merged.